activerecord/lib/active_record/connection_adapters/sqlite3/schema_definitions.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module ColumnMethods
+        def primary_key(name, type = :primary_key, **options)
+          if options.delete(:auto_increment) == true
+            type = :primary_key
```

Comment:
Shouldn't this just error if you have a type other than `:primary_key` but ask for `:auto_increment`?